### PR TITLE
Azure AI Studio Insights

### PR DESCRIPTION
**Pull request by sudivate**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

To access preview dashboard please visit Azure AI Studio resource in Azure Portal and append below as a query parameter to the URL, after *.azure.com and  open “AI Studio Insights” from the “Workbooks” menu under “Monitoring”  group menu from left pane.

`?feature.enableWorkbooks=true&feature.workbookGalleryBranch=sudivate%2Fgen-ai-insights&feature.includePreviewTemplates=true&`

OverView Tab
image

Session Overview Tab
image

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -187,6 +190,7 @@
 /gallery/workbook/microsoft.mobilenetwork-mobilenetwork-sites.json @microsoft/azure-private-mec-workbooks
 /gallery/workbook/microsoft.apimanagement-service.json @microsoft/azure-api-management
 /gallery/workbook/microsoft.experimentation-experimentworkspaces.json @microsoft/exp-devs
+/gallery/workbook/microsoft.machinelearningservices-workspaces @microsoft/gen-ai-insights
```
Comment (gardnerjr) [inline CODEOWNERS:193]: you added 2 gallery files, so you should have 2 gallery entries in CODEOWNERS

File: gallery/aistudio-insights/microsoft.machinelearningservices-workspaces.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
```
Comment (gardnerjr) [inline gallery/aistudio-insights/microsoft.machinelearningservices-workspaces.json:2]: reminder, this one won't actually *work* until the workbooks team actually adds this aistudio-insights as an official supported type in code, and i'm still waiting on the details for that from mail/teams.  given CCOA's and safedeploy its probably closing in on 10 days to be available if it happens today?

File: CODEOWNERS (hunk 1 above)
Comment (sudivate) [reply CODEOWNERS:193]: Done

**Review by gardnerjr — APPROVED** [review]
approving gallery/codeowners changes, haven't reviewed the actual template content aside from screenshots

**Pull request merged**